doc/design/fluid_dist/parameter_server.md

```diff
@@ -59,6 +59,13 @@ After converting:
      queue. It will block until the queue has the required number of
      tensors.
 
+### Sparse Update
+
+For an embedding layer, the gradient maybe be very sparse(upper 90% is zero) for each mini-batch.
+Fluid use [SelectedRows](../selected_rows.md) to support the sparse variable. Distributed training support `Sparse Update`,
```

Comment:
the sparse variable => sparse variables.

Reply:
support => supports

Reply:
Distributed training support `Sparse Update`, which sends a `SelectedRows` variable to the parameter server to run parameter updates.

Reply:
Done.